### PR TITLE
Added missing Articulation type ('other-articulation') and added parse for notes with type print-Object

### PR DESCRIPTION
**Pull request by nhatlee**
+ Some songs which have Articulation type with name 'other-articulation', currently the library still missing this type, so the app will crash if not parse this type(crash exception:`Invalid articulation type...`).
+ In music XML, some song have notes with type `print-object` must treat as rest notes. Ex:
```
<measure number="3" width="379">
      <print new-system="yes">
        <system-layout>
          <system-distance>147</system-distance>
        </system-layout>
      </print>
      <note print-object="no">
        <rest/>
        <duration>1</duration>
        <voice>1</voice>
      </note>
      <barline location="right">
        <bar-style>none</bar-style>
      </barline>
    </measure>
```

### CODE REVIEW

**Comment by nhatlee** [comment]
With the commit to parse `print-object` will fixed some kind of bugs like that:
img_0168
After added missing parse for `print-object`. The music xml will display properly:
img_0169

File: src/mxml/SpanFactory.cpp [hunk 1]
```diff
@@ -271,8 +271,8 @@ void SpanFactory::build(const dom::Chord* chord) {
 void SpanFactory::build(const dom::Note* note) {
     assert(note->rest);
 
-    if (!note->printObject)
-        return;
+//    if (!note->printObject)
+//        return;
```
Comment (aleph7) [inline src/mxml/SpanFactory.cpp]: Let's delete this.

File: src/mxml/geometry/factories/DirectionGeometryFactory.cpp [hunk 2]
```diff
@@ -489,5 +496,45 @@ void DirectionGeometryFactory::buildCoda(const MeasureGeometry& measureGeom, con
...
+    
+void DirectionGeometryFactory::buildBracket(const MeasureGeometry&  measureGeom, const dom::Direction& direction) {
+        using dom::Bracket;
+        bool isPage = measureGeom.scoreProperties().layoutType() == ScoreProperties::LayoutType::Page;
+        const Bracket& bracket = dynamic_cast<const Bracket&>(*direction.type());
+        if (bracket.type() == dom::kStart) {
+            auto pair = pullPedalStart(measureGeom, direction);
+            if (pair.first) {
+                //It seem this case never happen with Bracket
+                printf("If see this line in the log then we need to review code and handle it!!!!\n");
```
Comment (aleph7) [inline src/mxml/geometry/factories/DirectionGeometryFactory.cpp]: Let's use `assert` instead.

File: src/mxml/geometry/factories/MeasureGeometryFactory.cpp [hunk 3]
```diff
@@ -335,8 +335,8 @@ void MeasureGeometryFactory::placeChord(ChordGeometry* chordGeom) {
 
 void MeasureGeometryFactory::buildRest(const Note* note) {
     assert(note->rest);
-    if (!note->printObject)
-        return;
+//    if (!note->printObject)
+//        return;
```
Comment (aleph7) [inline src/mxml/geometry/factories/MeasureGeometryFactory.cpp]: Let's delete this.

**Pull request merged**